app/domain/authentication/authn_jwt/restriction_validators/fetch_mandatory_claims.rb

```diff
@@ -0,0 +1,73 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module RestrictionValidators
+      # Fetch the mandatory claims from the JWT authenticator policy which enforce 
+      # definition of annotations keys on JWT hosts 
+      FetchMandatoryClaims = CommandClass.new(
+        dependencies: {
+          resource_class: ::Resource,
+          fetch_required_secrets: ::Conjur::FetchRequiredSecrets.new,
+          parse_mandatory_claims: ::Authentication::AuthnJwt::InputValidation::ParseMandatoryClaims.new,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters]
+      ) do
+        def call
+          fetch_mandatory_claims
+        end
+
+        private
+
+        def fetch_mandatory_claims
+          @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingMandatoryClaims.new)
+          if !mandatory_claims_resource_exists?
+            @logger.debug(LogMessages::Authentication::AuthnJwt::NotConfiguredMandatoryClaims.new)
+            return empty_mandatory_claims
+          else
```

Comment:
what is the purpose of "else"? we will exit the function with the "return" when the if condition isn't mapped..

Reply:
fixed